CONTRIBUTING.md

```diff
@@ -78,7 +78,7 @@ Once the release is prepared, it'll go through a review:
 
 Finally, an [FVM "owner"](https://github.com/orgs/filecoin-project/teams/fvm-crate-owners/members) will:
 
-1. Merge the release PR to master.
+1. Merge the release PR to master (this will trigger GitHub release/tag creation if the workspace version in the root `Cargo.toml` was modified).
```

Comment:
Is this complete?  Are there steps from https://github.com/filecoin-project/ref-fvm/pull/2027#issuecomment-2254402698 we need to include?  I don't see discussion about opening a PR which creates a draft Release and that when we merge the PR we get a published Release.

And are the comments about an FVM owner still needing to run `cargo publish` accurate?  (or are we only doing a dryrun as part of CI?)